server/settings/include_handles.py

```diff
@@ -0,0 +1,30 @@
+from ayon_server.settings import (
+    BaseSettingsModel,
+    SettingsField,
+    task_types_enum,
+)
+
+
+class IncludeByTaskTypeModel(BaseSettingsModel):
+    task_type: list[str] = SettingsField(
+        default_factory=list,
+        title="Task types",
+        enum_resolver=task_types_enum
+    )
+    include_handles: bool = SettingsField(True, title="Include handles")
+
+
+class IncludeHandlesModel(BaseSettingsModel):
+    include_handles_default: bool = SettingsField(
+        True, title="Include handles by default"
+    )
+    per_task_type: list[IncludeByTaskTypeModel] = SettingsField(
```

Comment:
Couldn't this be profiles?

Reply:
Done with https://github.com/ynput/ayon-blender/pull/28/commits/6534f1c5f390359f36ec3ed1bb8e0b0560ce4855